sonic-slave-buster/Dockerfile.j2

```diff
@@ -417,3 +419,7 @@ RUN apt-get install -y docker-ce=18.06.3~ce~3-0~debian docker-ce-cli=18.06.3~ce~
 {%- endif %}
 RUN echo "DOCKER_OPTS=\"--experimental --storage-driver=vfs\"" >> /etc/default/docker
 RUN update-alternatives --set iptables /usr/sbin/iptables-legacy
+
+# Install swi tools
+RUN git clone https://github.com/aristanetworks/swi-tools.git /usr/share/swi_tools
+RUN cd /usr/share/swi_tools && python setup.py install
```

Comment:
Can we fix the commit ID?

Reply:
Fixed. Add the commit ID in the command line, right?